server/data/carriers_of_hope.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE members (
+  id          SERIAL PRIMARY KEY,
```

Comment:
if we will use uuid then we need to add it in our database this my opinion; shall we ask it Sami!